localisation/00_zuwc_united_localisation.csv

```diff
@@ -1119,22 +1119,22 @@ EVTDESC_WCPOD_5014;This morning you were told that your [GetFromRelation] [From.
 EVTOPTB_WCPOD_5014_glad;Now we are on the same side.;;Nun sind wir auf derselben Seite.;;;;;;;;;;;x
 EVTOPTB_WCPOD_5014_shocked;What� How?;;Was? Wie?;;;;;;;;;;;x
 EVTOPTB_WCPOD_5014_undead;We shared the same fate.;;Uns ist das Selbe widerfahren.;;;;;;;;;;;x
-EVTDESC_WCUND_350;While your prisoner [target_prisoner.GetFirstName] was tied to the surgical table, you prepared everything for the long-awaited operation. [target_prisoner.GetFirstName] attempted to break the restraints or at least scream, but it was all in vain. When my servants made the first cuts, the exhausting groans of the victim echoed in the operating room, which only ceased when [target_prisoner.GetFirstName] lost [target_prisoner.GetHerHis] consciousness from pain and shock.;;W�hrend [target_prisoner.GetEuerEure] Gefangene[target_prisoner.GetrMale] [target_prisoner.GetFirstName] an den Tisch gefesselt ist, bereitetet Ihr alles f�r die lang erwartete Operation vor. [target_prisoner.GetFirstName] versucht sich loszurei�en oder zumindest zu schreien, aber es ist alles vergeblich. Als Eure Diener die ersten Schnitte machen, hallt der Operationsraum vom herzzerrei�enden St�hnen des Opfers wider, das erst aufh�rte, als [target_prisoner.GetFirstName] vor Schmerzen und Schock das Bewusstsein verliert.;;;;;;;;;;;x
-EVTOPTA_WCUND_350;It will take some time.;;Es wird eine Weile dauern.;;;;;;;;;;;x
+EVTDESC_WCUND_350_abomination;While your prisoner [target_prisoner.GetFirstName] was tied to the surgical table, you prepared everything for the long-awaited operation. [target_prisoner.GetFirstName] attempted to break the restraints or at least scream, but it was all in vain. When my servants made the first cuts, the exhausting groans of the victim echoed in the operating room, which only ceased when [target_prisoner.GetFirstName] lost [target_prisoner.GetHerHis] consciousness from pain and shock.;;W�hrend [target_prisoner.GetEuerEure] Gefangene[target_prisoner.GetrMale] [target_prisoner.GetFirstName] an den Tisch gefesselt ist, bereitet Ihr alles f�r die lang erwartete Operation vor. [target_prisoner.GetFirstName] versucht sich loszurei�en oder zumindest zu schreien, aber es ist alles vergeblich. Als Eure Diener die ersten Schnitte machen, hallt der Operationsraum vom herzzerrei�enden St�hnen des Opfers wider, das erst aufh�rt, als [target_prisoner.GetFirstName] vor Schmerzen und Schock das Bewusstsein verliert.;;;;;;;;;;;x
```

Comment:
Could switch the first sentence around to "You prepared everything for the long-awaited operation while your prisoner [target_prisoner.GetFirstName] was tied to the surgical table."
"victim echoed in the operating room" changed to "victim echoed throughout the operating room"

Is all of this supposed to be past tense or should it be present tense, because the other events related to this seem to switch between the two, I need to know before suggesting more changes to this file.

Reply:
> Could switch the first sentence around to "You prepared everything for the long-awaited operation while your prisoner [target_prisoner.GetFirstName] was tied to the surgical table."

Does that really change a thing?

Reply:
This is more of a present/past tense problem, it's inconsistent, knowing which one it should be would help in making actual confident changes.

Reply:
but, it's the same tense in the original one...
Was tied .... you prepared... --> You prepared... was tied

Reply:
EVTDESC_WCUND_355_abomination, so is 370 is in present tense, this one is in past tense, unless the event (350) is being told from 355 and 370's perspective, I consider it inconsistent.